src/EnergyPlus/Data/EnergyPlusData.cc

```diff
@@ -570,11 +571,15 @@ void EnergyPlusData::clear_state()
 
 void EnergyPlusData::init_state(EnergyPlusData &state)
 {
+    if (this->init_state_called) return;
+    this->init_state_called = true;
     // The order in which we do this matters.  We're going to try to
     // do this in "topological" order meaning the first to go are the
     // objects that do not reference any other objects, like fluids,
     // schedules, curves, etc.
-    this->dataFluidProps->init_state(state);
+    this->dataSimulationManager->init_state(state); // GetProjectData
+    this->dataFluidProps->init_state(state);        // GetFluidPropertiesData
+    this->dataPsychrometrics->init_state(state);    // InitializePsychRoutines
 
```

Comment:
I was going to suggest that you do this (i.e., start sorting the `init_state` calls) but you are already doing it.

Reply:
I'm not quite 100% convinced we can come up with a single order that works for every single simulation configuration.  I'm happy to be shown that we actually can, but I expected the init calls to be a bit more surgical, with individual inits calling out to child inits.  Anyway, let's see where this goes.